network/network.go

```diff
@@ -75,6 +75,7 @@ type NetworkInfo struct {
 	IPAMType                      string
 	ServiceCidrs                  string
 	IsIPv6Enabled                 bool
+	NICType                       string
```

Comment:
are we still using NetworkInfo? i thought this structure is not used

Reply:
removed NICType from NetworkInfo